.github/ISSUE_TEMPLATE/setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Setup
+about: Prepare for development with some basic project setup.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`). If child tickets are created for any line-item,
+please update this ticket description to include references to those tickets.
+
+- [ ] Create a README.
+- [ ] Set up Tugboat.
+- [ ] Set up Dependabot.
+- [ ] Set up PHP code style checks.
+- [ ] Set up front-end linting.
+- [ ] Set up GitHub Actions to run code style checks on YML, shell scripts, and
+      execute PHP checks/front-end linting tools.
+- [ ] Set up `settings.php` file infrastructure.
+- [ ] Set up Composer file.
+- [ ] Set up front-end tooling.
+- [ ] Set up Lando file.
+- [ ] Set up a production environment.
+- [ ] Set up nightly database backups.
+- [ ] Set up development servers.
```

Comment:
What do we mean by this?

Reply:
We can probably remove this since we don't really use development servers now.

Reply:
Removed.